proposals/shared-everything-threads/Overview.md

```diff
@@ -448,6 +448,20 @@ may be observable if it causes finalizers registered in the host to be fired.
 
 [wait-notify]: https://github.com/WebAssembly/threads/blob/main/proposals/threads/Overview.md#wait-and-notify-operators
 
+### Spinlock Relaxation: `nop_pause`
```

Comment:
Any reason why we can't just call the instruction `pause`?

Reply:
Good idea, that seems cleaner.